net/tinc/Makefile

```diff
@@ -8,24 +8,30 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tinc
-PKG_VERSION:=1.0.36
-PKG_RELEASE:=3
+PKG_VERSION:=1.1-git
+PKG_RELEASE=$(PKG_SOURCE_VERSION)-1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://www.tinc-vpn.org/packages
-PKG_HASH:=40f73bb3facc480effe0e771442a706ff0488edea7a5f2505d4ccb2aa8163108
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=http://tinc-vpn.org/git/tinc
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=3ee0d5dd
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
+PKG_MIRROR_HASH:=fe57dcce5c17ec2cea96fefc69c5af977df7d457f1ae7c4775e00a21878a2c19
 
 PKG_CPE_ID:=cpe:/a:tinc:tinc
 
 PKG_BUILD_PARALLEL:=1
+PKG_CONFIG_DEPENDS:=zlib lzo openssl
+PKG_BUILD_DEPENDS:=zlib lzo openssl
+PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
 
 define Package/tinc
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+liblzo +libopenssl +kmod-tun +zlib
+  DEPENDS:=+kmod-tun +liblzo +libopenssl +librt +zlib
   TITLE:=VPN tunneling daemon
   URL:=http://www.tinc-vpn.org/
   MAINTAINER:=Saverio Proto <zioproto@gmail.com>
```

Comment:
Please update the MAINTAINER with your details.

Reply:
I am not 100% sure how you get merge rights on github for this package. I dont think the line in the Makefile is enough. You would have to follow up on the OpenWrt Developers mailing list to ask what is the actual procedure for this today.
Thanks for stepping in as Maintainer of Tinc